src/main/java/org/broadinstitute/consent/http/db/mapper/DataAccessRequestMapper.java

```diff
@@ -25,8 +25,15 @@ public DataAccessRequest map(ResultSet resultSet, StatementContext statementCont
     if (hasColumn(resultSet, "dar_code")) {
       dar.setDarCode(resultSet.getString("dar_code"));
     }
-    dar.setParentId(resultSet.getInt("parent_id"));
-    dar.setUserId(resultSet.getInt("user_id"));
+
+    if (hasNonZeroColumn(resultSet,"parent_id")) {
+      dar.setParentId(resultSet.getInt("parent_id"));
+    }
+
+    if (hasNonZeroColumn(resultSet, "user_id")) {
```

Comment:
Drive by to be consistent but database schema requires a non-empty value for this column so we don't run into this case.